mappings/net/minecraft/test/TestContext.mapping

```diff
@@ -74,7 +93,8 @@ CLASS net/minecraft/unmapped/C_ncfucati net/minecraft/test/TestContext
 		ARG 1 pos
 		ARG 2 item
 	METHOD m_gtojqzaf getTick ()J
-	METHOD m_gufdcbtu assertAtTickContainerContents (JLnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_vorddnax;)V
+	METHOD m_gufdcbtu expectFutureContainerWith (JLnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_vorddnax;)V
```

Comment:
this feels a bit too mojang, could we go with inventory instead of container?

Reply:
Not sure, the class it checks is currently called `LootableContainerBlockEntity` in quilt mappings

Reply:
ah, that works then. I should have taken more time to check on the context :p